en/guide/missions.md

```diff
@@ -1,10 +1,27 @@
 # Missions
 
-The DroneCore Mission API allows you to create, upload, run, pause, restart, jump to item in, and track missions. Missions can have multiple "mission items", each which may specify a position, altitude, fly-through behaviour, camera action, gimbal position, and the speed to use when traveling to the next position.
+The DroneCore Mission API allows you to create, upload, download, run, pause, restart, jump to item in, and track missions. Missions can have multiple "mission items", each which may specify a position, altitude, fly-through behaviour, camera action, gimbal position, and the speed to use when traveling to the next position.
 
-> **Note** The API enables a small but useful subset of the mission commands supported by PX4 (and the MAVLink specification). For example, it does not currently support "repeat", takeoff, return to land etc.
+Missions are *managed* though the [Mission](../api_reference/classdronecore_1_1_mission.md) class, which communicates with the device to upload mission information and run, pause, track the mission progress etc. The mission that is uploaded to the vehicle is defined as a vector of [MissionItem](../api_reference/classdronecore_1_1_mission_item.md) objects.
 
-Missions are *managed* though the [Mission](../api_reference/classdronecore_1_1_mission.md) class, which communicates with the device to upload mission information and run, pause, track the mission progress etc. The mission that is uploaded to the vehicle is defined as a vector of [MissionItem](../api_reference/classdronecore_1_1_mission_item.md) objects. 
+
+## Supported Mission Commands {#supported_mission_commands}
+
+The [MissionItem](../api_reference/classdronecore_1_1_mission_item.md) class abstracts a small but useful subset of the mission commands supported by PX4 (and the MAVLink specification):
+
+The supported set is:
+* [MAV_CMD_NAV_WAYPOINT](http://mavlink.org/messages/common#MAV_CMD_NAV_WAYPOINT)
+* [MAV_CMD_DO_CHANGE_SPEED](http://mavlink.org/messages/common#MAV_CMD_DO_CHANGE_SPEED)
+* [MAV_CMD_DO_MOUNT_CONTROL](http://mavlink.org/messages/common#MAV_CMD_DO_MOUNT_CONTROL)
+* [MAV_CMD_IMAGE_START_CAPTURE](http://mavlink.org/messages/common#MAV_CMD_IMAGE_START_CAPTURE)
+* [START_PHOTO_INTERVAL](http://mavlink.org/messages/common#START_PHOTO_INTERVAL)

```

Comment:
`START_PHOTO_INTERVAL` and `TAKE_PHOTO` are part of `MAV_CMD_IMAGE_START_CAPTURE`, so not actually mavlink commands.